extras/PackageMaintenance.R

```diff
@@ -82,7 +82,7 @@ connectionDetails <- DatabaseConnector::createConnectionDetails(dbms = "postgres
                                                                 user = Sys.getenv("CDM5_POSTGRESQL_USER"),
                                                                 password = URLdecode(Sys.getenv("CDM5_POSTGRESQL_PASSWORD")),
                                                                 server = Sys.getenv("CDM5_POSTGRESQL_SERVER"))
-cohortDiagnosticsSchema <- Sys.getenv("CDM5_POSTGRESQL_COHORT_DIAGNOSTICS_SCHEMA")
+cohortDiagnosticsSchema <- "cohort_diagnostics"
```

Comment:
@keesvanbochove  why not check if the environment variable exists. If it returns NULL then pass the hardcoded value as a fall back default value?